### ISSUE TITLE
vr::VRDriverInput() can be NULL in IServerTrackedDeviceProvider::Init

### ISSUE THREAD

**Issue by gb2111** [ValveSoftware/openvr#1079]
as stated in title. 
i was looking for some issues with my code and finally found that vr::VRDriverInput() can be null calling from implementation of driver's IServerTrackedDeviceProvider::Init.
It can happen very occasionally but still.

Regards.

**Comment by spayne** [comment]
I haven't seen that particular problem.  Just checking: Are you calling VR_INIT_SERVER_DRIVER_CONTEXT inside your IServerTrackedDeviceProvider::Init override?  I think you should be...

**Comment by gb2111** [comment]
yes. i am calling that. it happens only for minor fraction of users. 
but it was quite easy to debug and that was eventually the situation.
unfortunately I can't provide more info to reproduce.
